sdk/src/main/resources/log4j2.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="WARN" monitorInterval="30">
+<Configuration status="WARN" monitorInterval="30" shutdownHook="disable">
```

Comment:
Is "disable" a valid value? I say "true|false" in the docs

Reply:
I've tried both but this is the only value that works in my case

Reply:
right, it is specified to be set as disabled.